zerberus/zone.cpp

```diff
@@ -42,13 +42,13 @@ Zone::~Zone()
 //   match
 //---------------------------------------------------------
 
-bool Zone::match(Channel* c, int k, int v, Trigger et, double rand)
+bool Zone::match(Channel* c, int k, int v, Trigger et, double rand, int cc, int ccVal)
       {
 
-      if ((k >= keyLo)
-         && (k <= keyHi)
-         && (v >= veloLo)
-         && (v <= veloHi)
+      if ((k >= keyLo || et == Trigger::CC)
```

Comment:
it seems that CC just triggers  a sample and doesn't stop it after afterwards (as long it is not stopped by a group)
I know Trigger::CC is not in the spec - but at least for me it makes sense to implement it in this way!